resources/styles/components/task-plan/reading.less

```diff
@@ -25,13 +29,35 @@
 ul.selected-reading-list {
   list-style: none;
   padding-left: 0;
-  margin-top: 2em;
+  margin-top: 1em;
 
   li {
     border-bottom: 1px solid @tutor-neutral-lighter;
+    animation: fadein 2.5s;
+    -moz-animation: fadein 2.5s;
+    -webkit-animation: fadein 2.5s;
+    -o-animation: fadein 2.5s;
+
+    @keyframes fadein {
+    from { opacity:0; }
+    to { opacity:1; }
+    }
+    @-moz-keyframes fadein {
+        from { opacity:0; }
+        to { opacity:1; }
+    }
+    @-webkit-keyframes fadein {
+        from { opacity:0; }
+        to { opacity:1; }
+    }
+    @-o-keyframes fadein {
+        from { opacity:0; }
+        to { opacity: 1; }
+    }
 
```

Comment:
I think we can drop the vendor prefixes for `animation` and `keyframes`.  They're both supported by ie10+  http://caniuse.com/#search=animation http://caniuse.com/#search=keyframes